src/ansi-c/ansi_c_internal_additions.cpp

```diff
@@ -89,6 +94,10 @@ const char clang_builtin_headers[]=
 #include "clang_builtin_headers.inc"
 ; // NOLINT(whitespace/semicolon)
 
+const char windows_builtin_headers[]=
+  "int __noop();\n"
```

Comment:
Should this have a file name as the others do?

Reply:
Seemed unnecessary luxury for two lines.

Reply:
One day those 2 lines will be 10 and no-one will notice that they should have inserted the file name. It's near zero cost right now, but it might prove costly later.